### PR TITLE
Update installing.md

### PR DESCRIPTION
**Pull request by Loki-101**
Added a small section about DNS since I do see that brought up in the Discord Server often enough to warrant a quick mention here.

### CODE REVIEW

File: wings/1.0/installing.md [hunk 1]
```diff
@@ -140,6 +140,16 @@ sudo wings --debug
 
 You may optionally add the `--debug` flag to run Wings in debug mode.
 
+### Configuring DNS (Usually not needed, refer back to this if your containers have no internet connection)
+Now that Wings has run successfully and you have gotten the green heart on your Nodes page, the wings config at '/etc/pterodactyl/config.yml' will have new values.
+One of those values is DNS, which by default will be 1.1.1.1 and 1.0.0.1
+For your containers to have internet, your DNS Servers in your wings config MUST match your DNS Servers on your host system.
```
Comment (parkervcp) [inline wings/1.0/installing.md:146]: The DNS in the container does not need to match the hosts DNS configuration. The place where this is an issue is for hosts that block the cloudflare ip addresses.

This is most likely due to old configurations or some thought that it makes things more secure.

File: wings/1.0/installing.md [hunk 2]
```diff
@@ -140,6 +140,16 @@ sudo wings --debug
...
+To view what DNS Servers your host system is using, you can use systemd-resolve, which typically will be on any system that uses systemd.
```
Comment (parkervcp) [inline wings/1.0/installing.md:147]: You should specify that `systemd-resolve --status` is the command to run if they want to see their DNS servers.

It's also important to know that not all distributions use `systemd-resolve` to manage their dns.

**Review by parkervcp — CHANGES_REQUESTED** [review]

**Comment by Loki-101** [comment]
Thank you. I think it's clearer now, and I've removed the included filters so just lists all the information, as I don't know how well the filters would work with multiple network interfaces in use.

**Comment by Boy132** [comment]
IMO this should be moved to the "Troubleshooting" section or to "Additional Configuration" because as parker mentioned not everyone has this problem.

**Comment by Loki-101** [comment]
Good point. I'll close this and make a request after moving this section to troubleshooting tomorrow.

**Pull request closed without merging**